ngraph/core/src/type/element_type.cpp

```diff
@@ -167,7 +167,15 @@ size_t element::Type::hash() const
 
 const std::string& element::Type::get_type_name() const
 {
-    return get_type_info_map().at(m_type).m_type_name;
+    try
+    {
+        return get_type_info_map().at(m_type).m_type_name;
```

Comment:
Please explain when it is needed? 
s_type_info_map mast always contain all data types.

Reply:
rollback: https://github.com/eshoguli/openvino/blob/es/lpt/lpt_to_ngraph_fixes2_with_master/ngraph/core/src/type/element_type.cpp#L170